app/next.config.js

```diff
@@ -1,9 +1,13 @@
 const pkg = require('../package.json');
+const path = require('path');
 
 module.exports = {
   env: {
     LIB_VERSION: pkg.version,
   },
+  sassOptions: {
+    prependData: `@import ${path.join(__dirname, './variables/_mixins.sass')}`,
```

Comment:
1. It's not good to create an extra folder on the project root. Move it to `src/styles/_mixins.sass`.
2. Also, this method not clear for development. I think it better to use manual `@export` for each file.